clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -171,6 +171,10 @@ class AutoScalingWorker {
     String launchConfigName = null
 
     if (shouldSetLaunchTemplate(asgConfig)) {
+      if (asgConfig.getCredentials().getEnvironment() == 'test' && dynamicConfigService.isEnabled("aws.features.launch-templates.ipv6", false)) {
```

Comment:
It'd be kinda nice to not bind the oss code directly against our environments. Perhaps a configuration option.

Even ... `aws.features.launch-templates.ipv6.${enviroment}` might work.

Reply:
Long term it will be enabled in all environments by default, but this is not something we want to force oss to do. This this is a good suggestion to prevent that.